model/project.go

```diff
@@ -1293,8 +1293,10 @@ func FindLatestVersionWithValidProject(projectId string) (*Version, *Project, er
 		"last good version for project '%s'", lastGoodVersion.Identifier)
 }
 
-func (bvt *BuildVariantTaskUnit) HasBatchTime() bool {
-	return bvt.CronBatchTime != "" || bvt.BatchTime != nil || bvt.Activate != nil
+// HasSpecificActivation returns if the build variant task specifies an overriding activation,
```

Comment:
The wording here was a little confusing for me to understand, can this be phrased to mention that the default is to unconditionally activate the task? Something like "returns if the build variant task specifies an activation condition that overrides the default to activate the task"?

Reply:
Tweaked, how is that?

Reply:
Perfect, thanks!